system_test/aest_sync_SUITE.erl

```diff
@@ -248,6 +286,89 @@ crash_and_continue_sync(Cfg) ->
             ?assertEqual(B1, B2)
     end.
 
+net_split_recovery(Cfg) ->
+    Length = 10,
+
+    setup_nodes([?NET1_NODE1, ?NET1_NODE2, ?NET2_NODE1, ?NET2_NODE2], Cfg),
+    start_node(net1_node1, Cfg),
+    start_node(net1_node2, Cfg),
+    start_node(net2_node1, Cfg),
+    start_node(net2_node2, Cfg),
+
+    %% Starts with a net split
```

Comment:
Since net1_node1 is connected to net2_node1 I don't see why this is building two different chains. In fact it starts with a joined network, as far as I can see

Reply:
`net1_node1` as `net2_node2` as a peer BUT the container is not connected to the `net2` network. So it will fail to connect to the other node and end-up forking the chain.